web/src/App.tsx

```diff
@@ -16,6 +16,8 @@ import { initSentry } from './utils/sentry'
 
 const GlobalStyle = createGlobalStyle`
   body {
+  position: relative;
```

Comment:
I think the indentation is off:
```suggestion
    position: relative;
```